src/consumer/runner.js

```diff
@@ -172,9 +174,14 @@ module.exports = class Runner {
         heartbeat: async () => {
           await this.consumerGroup.heartbeat({ interval: this.heartbeatInterval })
         },
+        /**
+         * Note that this method _does not_ heed the "autoCommit" option,
```

Comment:
This is also used by `eachMessage` to enforce the autoCommit policies.

https://github.com/tulios/kafkajs/blob/b78461a4798907379f91b1435d19362cf3b54d55/src/consumer/runner.js#L159